internal/cobraext/command.go

```diff
@@ -0,0 +1,68 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License
+// 2.0; you may not use this file except in compliance with the Elastic License
+// 2.0.
+
+package cobraext
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// CommandContext is the context in which an elastic-agent-changelog command runs.
+type CommandContext string
+
+const (
+	// ContextGlobal means the command runs in a global context, agnostic of any
+	// specific package.
+	ContextGlobal CommandContext = "global"
+
+	// ContextPackage means the command runs in the contexts of a specific package.
+	ContextPackage CommandContext = "package"
+)
```

Comment:
What is the purpose of `CommandContext` in this CLI?

Reply:
This might be useful in the future for providing the context when running in scripts etc. But you are correct, no need to include it at this point.